src/main/app/forms/validation/validators/isCountrySupported.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  registerDecorator,
+  ValidationArguments,
+  ValidationOptions,
+  ValidatorConstraint,
+  ValidatorConstraintInterface
+} from 'class-validator'
+import { Country } from 'app/common/country'
+
+import { PostcodeInfoClient, PostcodeInfoResponse } from '@hmcts/postcodeinfo-client'
+import * as config from 'config'
+import { request } from 'client/request'
+
+const postcodeClient = new PostcodeInfoClient(config.get<string>('postcodeLookup.apiKey'), request)
+
+@ValidatorConstraint()
+export class CheckCountryConstraint implements ValidatorConstraintInterface {
+
+  async validate (value: any | string, args?: ValidationArguments): Promise<boolean> {
```

Comment:
Please add unit tests for this validate method that tests the following:
- undefined value
- null value
- empty value
- invalid postcode
- not support postcode 
- supported postcode

Reply:
I've added the tests now

Reply:
@geek-soft do we want to test `null`?